src/main/java/com/karankumar/bookproject/ui/goal/ComponentGoalEvent.java

```diff
@@ -0,0 +1,19 @@
+package com.karankumar.bookproject.ui.goal;
```

Comment:
In line with our style guide, could you please attach our [copyright notice](https://github.com/Project-Books/book-project/blob/master/docs/COPYRIGHT) to the start of this file?